nft/nfts.yaml

```diff
@@ -992,7 +992,7 @@ components:
         type: string
         enum:
           - seaport
-        default: seaport
```

Comment:
what's the default here now?

Reply:
Default is we return for every marketplace.